### PR TITLE
Travis: fix speed up for noi18n + protractor

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Following https://github.com/jhipster/generator-jhipster/pull/8285
Ping @hdurix 

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/shared/shared-libs.module.ts.ejs [hunk 1]
```diff
@@ -31,6 +31,7 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
         NgJhipsterModule.forRoot({
             // set below to true to make alerts look like toast
             alertAsToast: false,
+            alertTimeout: 5,
```
Comment (deepu105) [inline generators/client/templates/angular/src/main/webapp/app/shared/shared-libs.module.ts.ejs]: Want this in milliseconds? Sorry I'm on my fone. If it indeed is millis then 5 is too low

Comment (pascalgrimaud) [reply generators/client/templates/angular/src/main/webapp/app/shared/shared-libs.module.ts.ejs]: oh indeed, I think it's millis ! So it should be 5000

**Comment by hdurix** [comment]
Why is it needed? What is the exception with noi18n?

**Comment by pascalgrimaud** [comment]
@hdurix : it's for noi18n project.

For the `shared-libs.module.ts.ejs` file, JHipster will generate:

```typescript
NgJhipsterModule.forRoot({
    // set below to true to make alerts look like toast
    alertAsToast: false,
}),
```

After the generation, the `lint` will change to:

```typescript
NgJhipsterModule.forRoot({
    // set below to true to make alerts look like toast
    alertAsToast: false
}),
```

So the replace won't work, because of missing char `,` :

```
sed -e 's/alertAsToast: false,/alertAsToast: false, alertTimeout: 1,/1;' src/main/webapp/app/shared/shared-libs.module.ts > src/main/webapp/app/shared/shared-libs.module.ts.sed
```

**Pull request merged**